extensions/amp-accordion/1.0/accordion.js

```diff
@@ -72,8 +81,73 @@ export function Accordion({
     });
   }, [expandSingleSection]);
 
+  useImperativeHandle(
+    ref,
+    () => {
+      return {
```

Comment:
inline

Reply:
Thanks, Dima, updated in latest revision!